geo_to_hca/geo_to_hca.py

```diff
@@ -0,0 +1,301 @@
+# --- core imports
```

Comment:
Why is this a new file and not a rename?

Reply:
I have no idea. I did a rename and directory structure change. That's all.